packages/components/src/internal/app/utils.ts

```diff
@@ -164,6 +165,10 @@ export function isSampleManagerEnabled(moduleContext?: any): boolean {
     return (moduleContext ?? getServerContext().moduleContext)?.samplemanagement !== undefined;
 }
 
+export function isSampleManagerProfessionalEnabled(moduleContext?: any): boolean {
+    return isSampleManagerEnabled(moduleContext) && isWorkflowEnabled(moduleContext)
```

Comment:
Professional provides workflow + assay + ELN. Checking for workflow being enabled in seemed sufficient since the others require their respective modules, but this function is not actually used so I'll get rid of it.